app/assets/stylesheets/application.css.scss

```diff
@@ -202,3 +202,16 @@ main {
   color: white;
   box-shadow: 1px 1px 10px 1px rgba(0,0,0,0.7);
 }
+
+.town_table td{
+  border: none;
+  border-collapse: collapse;
+  border-spacing: 0px;
+  padding-bottom: 0px
+}
+
+.town_table img.responsive-img {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `town_table` should be written in lowercase with hyphens